src/cephadm/cephadm

```diff
@@ -2267,6 +2268,17 @@ def command_inspect_image():
 
 ##################################
 
+def is_ipv6(address):
+    # type: (str) -> bool
+    if address.startswith('[') and address.endswith(']'):
+        address = address[1:-1]
+    try:
+        return ipaddress.ip_address(address).version == 6
+    except ValueError:
+        logger.warning("Address: {} isn't a valid IP address".format(address))
+        return False
```

Comment:
this function is trivially testable in pytest. Do you want to add something like

```python
def test_is_ipv6():
    assert is_ipv6("[::1]")
    assert not is_ipv6("127.0.0.1")
```
?

Reply:
yup, great idea, I'll also add '[]' etc. ta.

Reply:
I mean non []